src/shared/components/Animations/ListItem.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+import './list-item.less';
+
+export interface ListItemProps {
+  label: string;
+  key: string;
+  description?: string;
+  detail?: React.ReactComponentElement<any>;
+  action?: React.ReactComponentElement<any>;
+  onClick?(): void;
```

Comment:
`onClick` should probably return the `id` of the element that was selected?

Reply:
that's a great idea